.github/workflows/ci.yml

```diff
@@ -41,11 +41,12 @@ jobs:
     - name: Install package
       run: uv pip install --system -e .[tests]
 
+    - name: Run unit tests
```

Comment:
Unit tests take only couple seconds so lets run them first. The "Smoke tests" now take more than a minute since running the `field_envelope.py` is (understandably) quite slow.